### PR TITLE
VPESG Integration

### PR DESCRIPTION
**Pull request by SunithaGudisagarIBM**
### Community Note

* Please vote on this pull request by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original pull request comment to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for pull request followers and do not help prioritize the request

Relates OR Closes #0000

Output from acceptance testing:

```
$ make testacc TEST=./ibm TESTARGS='-run=TestAccIBMISVirtualEndpointGateway_Basic_SecurityGroups'

...
```
Screenshot 2021-12-01 at 6 54 11 PM

### CODE REVIEW

**Comment by ujjwal-ibm** [comment]
include vpc-go-sdk changes. It is showing soflayer changes @SunithaGudisagarIBM

File: website/docs/r/is_virtual_endpoint_gateway.html.markdown [hunk 1]
```diff
@@ -77,6 +81,7 @@ Review the argument references that you can specify for your resource.
   **NOTE**: `id` and `subnet` are mutually exclusive.
 
 - `resource_group` - (Optional, Forces new resource, String) The resource group ID.
+- `security_groups` - (Optional, list) The security groups to use for this endpoint gateway.
```
Comment (sumitkumartiwari) [inline website/docs/r/is_virtual_endpoint_gateway.html.markdown:84]: Add in the description - If unspecified, the VPC's default security group is used.

File: website/docs/r/is_virtual_endpoint_gateway.html.markdown [hunk 2]
```diff
@@ -77,6 +81,8 @@ Review the argument references that you can specify for your resource.
   **NOTE**: `id` and `subnet` are mutually exclusive.
 
 - `resource_group` - (Optional, Forces new resource, String) The resource group ID.
+- `security_groups` - (Optional, list) The security groups to use for this endpoint gateway. If unspecified, the VPC's default security group is used.
+    **NOTE:** either of ibm_is_security_group_target resource or security_groups attribute should be used, can't be user together.
```
Comment (sumitkumartiwari) [inline website/docs/r/is_virtual_endpoint_gateway.html.markdown:85]: **NOTE:** either of `ibm_is_security_group_target` resource or `security_groups` attribute should be used, both can't be use together.

**Comment by SunithaGudisagarIBM** [comment]
https://github.com/IBM-Cloud/terraform-provider-ibm/pull/3488

**Pull request closed without merging**